packages/core/src/browser/view-container.ts

```diff
@@ -1234,7 +1236,19 @@ export class ViewContainerPart extends BaseWidget {
         };
     }
 
+    protected handleResize(): void {
+        const handleMouseEnter = () => {
+            this.node?.classList.add('no-pointer-events');
+            setTimeout(() => {
+                this.node?.classList.remove('no-pointer-events');
+                this.node?.removeEventListener('mouseenter', handleMouseEnter);
+            }, 1000);
```

Comment:
I feel like one second is a very long time until the icons appear again. Something like `100` ms is probably more appropriate and should prevent the flickering as well.

Reply:
Good point, one second is indeed way too long. I changed it to `100ms` and it seems to work well.